radicle-node/src/test/environment.rs

```diff
@@ -196,10 +195,13 @@ impl<G: Signer + cyphernet::Ecdh> NodeHandle<G> {
             if remaining.is_empty() {
                 break;
             }
-            tries += 1;
-            thread::sleep(Duration::from_millis(100));
+            events
+                .wait(
+                    |e| matches!(e, Event::SeedDiscovered { .. }),
+                    time::Duration::from_secs(9),
+                )
+                .unwrap();
```

Comment:
Not really but I tend to use multiples of 3 and 6 for timeouts because they divide well into minutes and hours.

Reply:
I think I had it at 9 because I was debugging something; but I could set it back to 6 for eg.

Reply:
Oh hmm. I mean the thing is the number will only be hit on failure, and it's important to *not* have false negatives (ie. test fails because timeout is too low). So basically the number should be very unlikely to hit when the software is working as expected. The downside is that you'll have some tests hang instead of erroring immediately.

One pro-tip on this is that `cargo-nextest` tells you which tests have been canceled when you Ctrl-C

Reply:
Ya, that's why I was thinking it could be a scalable constant based on the expected routes, e.g. `Duration::from_sec(1 * routes.len())`

But I get ya, it can always be notched down when debugging :+1:

Reply:
Yeah though I don't think the number of routes would really make a difference in how long it takes to converge...

Reply:
Perhaps the number of nodes? We'd have to experiment with this.